src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimSpecCallbackHandler.java

```diff
@@ -369,6 +372,18 @@ private CallbackResponse submitClaim(CallbackParams callbackParams) {
             dataBuilder.respondent2Represented(respondent2Represented);
         }
 
+        if (caseData.getRespondent1Represented() == NO
```

Comment:
We need to check what needs to happen when its Represented but not registered.

Reply:
After confirmation with Aaron, the case will goes offline and no pin and post needed for represented but not registered